Contents/Code/plex_api_helper.py

```diff
@@ -27,7 +27,7 @@
 from plexapi.utils import reverseSearchType
 
 # local imports
-from constants import contributes_to, guid_map
+from constants import contributes_to, guid_map, media_type_dict
```

Comment:
Need to add this `constants.py`

```py

media_type_dict = dict(
    art=dict(
        method=lambda item: item.uploadArt,
        type='art',
        name='art',
        themerr_data_key='art_url',
        remove_pref='bool_remove_unused_art',
    ),
    posters=dict(
        method=lambda item: item.uploadPoster,
        type='posters',
        name='poster',
        themerr_data_key='poster_url',
        remove_pref='bool_remove_unused_posters',
    ),
    themes=dict(
        method=lambda item: item.uploadTheme,
        type='themes',
        name='theme',
        themerr_data_key='youtube_theme_url',
        remove_pref='bool_remove_unused_theme_songs',
    ),
)
```

I can take care of this if needed.

Reply:
fixed